ngraph/core/include/ngraph/op/deformable_psroi_pooling.hpp

```diff
@@ -96,7 +97,7 @@ namespace ngraph
                 int64_t m_output_dim;
                 float m_spatial_scale;
                 int64_t m_group_size = 1;
-                std::string m_mode = "bilinear";
```

Comment:
This initial value was introduced with PR https://github.com/openvinotoolkit/openvino/pull/3620 by mistake. It comes from PSROIPooling but should be `bilinear_deformable` constructor defaults are correct so it shouldn't break compatibility.